resolver_service/views.py

```diff
@@ -1,33 +1,33 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
-import sys
-import os
-
-PROJECT_HOME = os.path.abspath(os.path.join(os.path.dirname(__file__), '.'))
```

Comment:
we should be able to do without these, use abslute imports (btw I have renamed the package from 'service' to 'resolver_service' - it is important to occupy a namespace that belongs to you; not a generic 'service')

- also, use either absolute imports 'resolver_service.app' or relative '.app' - but don't use 'import app' because that could find some other module called 'app'

Reply:
I have renamed the package from 'service' to 'resolver_service'
actually that is going to confuse me, I am going to keep it as service.